EMS/core-bundle/src/Resources/views/macros/data-field-type.html.twig

```diff
@@ -732,7 +732,9 @@
 					{% set displayOptions = dataField.fieldType.displayOptions %}
 					{% set iframeUrl = path('emsco_wysiwyg_styleset_iframe', {
 						'name': displayOptions.styles_set|default('default'),
-						'language': displayOptions.language|default('en'),
+						'language': dataField.formOptions.locale|default('en'),
+						'emsLink': dataField.formOptions.referredEmsId|default(null),
+						'field': dataField.fieldType.name
```

Comment:
The field name looks weak to me. I would prefer the field's path or the field id

Reply:
The name is unique inside a contentType structure. For example target all 'description' named field over all content types. But agree we also want to know the path, added an extra data-attribute 'field-path=[page][tabs][tab_fr][description_fr]'

Reply:
I disagree 2 (or more) fields may share the same name if at least one is nested

Reply:
Yes correct but now we have the name and the path available, It depends on the implementation.